api/app.py

```diff
@@ -21,7 +21,7 @@
 app.config['SQLALCHEMY_DATABASE_URI'] = f'postgresql+psycopg2://{DB_USER}:{DB_PASS}@{DB_HOST}:{DB_PORT}/{DB_NAME}'
 app.config['SQLALCHEMY_COMMIT_ON_TEARDOWN'] = True
 app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = True
-app.config["JWT_SECRET_KEY"] = os.getenv('SUPER_SECRET_KEY')
+app.config["JWT_SECRET_KEY"] = str(os.getenv('SUPER_SECRET_KEY'))
```

Comment:
the str should not be required. If you are wondering about the b in front of the string it tells python to turn it into a byte string. Returning this into a str could possibly break this

Reply:
If you were to take a look at `/home/matt/.local/share/virtualenvs/api-q5ilRnRI/lib/python3.6/site-packages/jwt/utils.py` on line 24 there is a function called `force_bytes` that is called when any of the JWT algorithms run.  The JWT secret config has to be a string or else all tests that rely on the JWT package will fail. (SignIn is a big one)